chrome/content/zotero/elements/itemPaneSection.js

```diff
@@ -140,10 +139,10 @@ class ItemPaneSectionElementBase extends XULElementBase {
 			return true;
 		}
 
-		if (!pendingFlag && renderFlag && newFlag === renderFlag) {
+		if (!isPending && isRendered) {
 			return true;
 		}
-		this[key] = newFlag;
+		this[key] = newDependencies;
```

Comment:
Will break some item pane tests. We used ti rely on the _renderItemID to busy wait for rendering a specific item in the tests. (Busy wait is not very elegant, though)